src/main/java/com/iexec/worker/feign/client/CoreClient.java

```diff
@@ -16,73 +16,64 @@
 
 package com.iexec.worker.feign.client;
 
-import java.util.List;
-
 import com.iexec.common.chain.WorkerpoolAuthorization;
 import com.iexec.common.config.PublicConfiguration;
 import com.iexec.common.config.WorkerModel;
 import com.iexec.common.notification.TaskNotification;
 import com.iexec.common.notification.TaskNotificationType;
 import com.iexec.common.replicate.ReplicateStatusUpdate;
 import com.iexec.common.security.Signature;
-
 import org.springframework.cloud.openfeign.FeignClient;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestHeader;
-import org.springframework.web.bind.annotation.RequestParam;
-
-import feign.FeignException;
+import org.springframework.web.bind.annotation.*;
 
+import java.util.List;
 
 @FeignClient(name = "CoreClient", url = "#{coreConfigurationService.url}")
 public interface CoreClient {
 
     @GetMapping("/version")
-    ResponseEntity<String> getCoreVersion() throws FeignException;
-
-    // worker
+    ResponseEntity<String> getCoreVersion();
 
+    //region /workers
     @GetMapping("/workers/challenge")
-    ResponseEntity<String> getChallenge(@RequestParam(name = "walletAddress") String walletAddress)
-            throws FeignException;
+    ResponseEntity<String> getChallenge(@RequestParam("walletAddress") String walletAddress);
 
     @PostMapping("/workers/login")
-    ResponseEntity<String> login(@RequestParam(name = "walletAddress") String walletAddress,
```

Comment:
Have you tested this in the spring's context? I think we had problems with RequestParam with no `name` field.

Reply:
I tested locally and it worked for me. Still the documentation is a bit unclear and the name property seemed to be unnecessary when the property name is the same as the parameter name.
I am doing some checks and will execute ITs to be sure.

Reply:
See 9f25dc9

Reply:
Fair enough!